src/api.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const myApi = axios.create({
+  baseURL: "https://backend-project-5gjj.onrender.com/api",
+});
+
+export const getAllArticles = (page) => {
+  return myApi
+    .get("/articles", {
+      params: {
+        page,
+      },
+    })
+    .then(({ data }) => {
+      return data.articles;
+    })
+    .catch((error) => {
```

Comment:
You'll want to catch this error in your component rather than here so that you can do something with it on the front-end